components/raftstore-v2/src/operation/command/admin/flashback.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+use engine_traits::{KvEngine, RaftEngine, RaftLogBatch};
+use fail::fail_point;
+use kvproto::{
+    raft_cmdpb::{AdminCmdType, AdminRequest, AdminResponse, RaftCmdRequest},
+    raft_serverpb::RegionLocalState,
+};
+use protobuf::Message;
+use raftstore::{coprocessor::RegionChangeReason, store::metrics::PEER_ADMIN_CMD_COUNTER, Result};
+
+use super::AdminCmdResult;
+use crate::{
+    batch::StoreContext,
+    fsm::ApplyResReporter,
+    raft::{Apply, Peer},
+};
+
+#[derive(Debug)]
+pub struct SetFlashbackState {
```

Comment:
Minor: struct name typically should not start with a Verb, can be renamed as FlashbackSetState. To be FlashbackState seems better.